tests/_files/classes.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Can we have a different number of each type, instead of 1, 1, 1 (maybe 1, 2 ,4)? Right now this test proves that each type is discovered as different from others, but counting final might be swapped with non-final.

Reply:
Great idea; just pushed a commit for that.

Reply:
I realized that in this case we should also deal with comments within the class declaration. So I've modified that in my latest commit.